src/datachain/lib/pytorch.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         tokenizer_kwargs: Optional[dict[str, Any]] = None,
         num_samples: int = 0,
         dc_settings: Optional[Settings] = None,
+        remove_prefetched: bool = False,
```

Comment:
I have added this setting for `PytorchDataset`, so prefetched items won't be removed after use by default. This is done so because this dataset could be used in multiple epochs, so I don't think it should be enabled by default.

Reply:
can it be determined by cache True / False though?

Reply:
`cache=True` will persist prefetched items to the cache. `PytorchDataset` can remove prefetched files on `.close()` or when it gets garbage collected. 

(I think we need a setting in Datachain itself for more control.)